Test/Api/Swagger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Test.Models;
+using System.Net;
+using Newtonsoft.Json;
+using Android.Util;
+
+namespace Test.Api
+{
+    class Swagger
+    {
+        private string root = "https://sdt-event-app.scapp.io/api";
+
+        private WebClient webClient = new WebClient();
+
+        private static Swagger instance;
+
+        public static Swagger Instance => instance ?? (instance = new Swagger());

```

Comment:
If you want to make this class is `Singeton`, you need to add the `private` `Contructor`

Reply:
Done